mmhuman3d/models/backbones/hrnet.py

```diff
@@ -766,3 +771,246 @@ def forward(self, x):
         xf = xf.mean(dim=(2, 3))
         xf = xf.view(xf.size(0), -1)
         return xf
+
+
+class PoseHighResolutionNetPyMAFX(BaseModule):
```

Comment:
Is it possible to inherit from `PoseHighResolutionNet` instead of `BaseModule`?

Reply:
In `PoseHighResolutionNet` ` __init__` need `final_conv_kernel` and  `stage1`, but we didn't define that parameters in `PoseHighResolutionNetPyMAFX`

Reply:
I think it is fine if `PoseHighResolutionNetPyMAFX` just ignore some parameters? We inherit from `PoseHighResolutionNet` and rewrite `__init___`, only resuing similar functions like `_make_transition_layer`.